examples.md

```diff
@@ -290,14 +291,33 @@ Replace `~/.cache/pip` with the correct `path` if not using Ubuntu.
       ${{ runner.os }}-pip-
 ```
 
+### Using pip to get cache location
+
+> Note: This requires pip 20.1+
+```yaml
+- name: Get pip cache dir
+  id: pip-cache
+  run: |
+    python -m pip install -U "pip>=20.1"
```

Comment:
Is it safe to recommend that users install pip just to get a cache directory? We could just keep the "Note: This requires pip 20.1+"  and leave out the install step.

Reply:
It would be safer to pin to `pip==20.1`, but then that would get out of date, so yeah, let's just leave out the command and let users make a conscious decision how to upgrade.

Updated!